allennlp/training/learning_rate_schedulers/learning_rate_scheduler.py

```diff
@@ -129,3 +136,79 @@ def __init__(
             eps=eps,
         )
         super().__init__(lr_scheduler)
+
+
+@LearningRateScheduler.register("constant")
+class ConstantLearningRateScheduler(_PyTorchLearningRateSchedulerWrapper):
+    """
+    Registered as a `LearningRateScheduler` with name "constant".  The "optimizer" argument does
+    not get an entry in a configuration file for the object.
+    """
+
+    def __init__(self, optimizer: Optimizer, last_epoch: int = -1) -> None:
+        lr_scheduler = get_constant_schedule(optimizer=optimizer, last_epoch=last_epoch)
+        super().__init__(lr_scheduler)
```

Comment:
It's fine to have this for completeness, but I'm pretty sure you could already get a constant learning rate if you just don't specify anything.